### PR TITLE
Test Case verifyMacBookIsAddedToProductComparison is Added

### PR DESCRIPTION
**Pull request by Daman1231**

### CODE REVIEW

File: src/main/java/com/naveenautomation/Pages/DesktopPage.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import com.naveenautomation.Browsers.ProxyDriver;
+
+public class DesktopPage extends Page {
+
+	public DesktopPage(WebDriver wd, boolean waitForPageToLoad) {
+		super(wd, waitForPageToLoad);
+		// TODO Auto-generated constructor stub
+	}
+
+	private static final String PAGE_URL = "/category&path=20";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/DesktopPage.java:15]: we can't have &path=20 in page URL

File: src/test/java/com/naveenautomation/tests/DesktopTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.testng.asserts.SoftAssert;
+
+import com.naveenautomation.Base.TestBase;
+import com.naveenautomation.Pages.AccountLoginPage;
+import com.naveenautomation.Pages.DesktopPage;
+import com.naveenautomation.Pages.MyAccountPage;
+
+public class DesktopTest extends TestBase {
+
+	AccountLoginPage accountLoginPage;
```
Comment (PivotITAcademy) [inline src/test/java/com/naveenautomation/tests/DesktopTest.java:15]: Add access modifiers

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]
Plz do the necessary changes

File: src/main/java/com/naveenautomation/Pages/DesktopPage.java (hunk 1 above)
Comment (Daman1231) [reply src/main/java/com/naveenautomation/Pages/DesktopPage.java]: Removed

File: src/test/java/com/naveenautomation/tests/DesktopTest.java (hunk 2 above)
Comment (Daman1231) [reply src/test/java/com/naveenautomation/tests/DesktopTest.java]: Access modifier added

**Review by PivotITAcademy — APPROVED** [review]
LGTM

**Pull request merged**